src/messages.cjs

```diff
@@ -88,7 +88,14 @@ Messages.prototype = {
       message = this._replacePlaceholders(rule, template, {});
     }
 
-    return message;
+    const data = {};
+    data.attribute = this._getAttributeName(rule.attribute);
+    data[rule.name] = data[rule.name] || rule.getParameters().join(',');
+
+    return {
+      message,
+      data,
```

Comment:
We're only testing the functionality of the `message` var. Is testing the `data` var not necessary or will this be added/tested at a later stage.